cmd/waku/server/rest/lightpush_api.yaml

```
@@ -0,0 +1,84 @@
+openapi: 3.0.3
+info:
+  title: Waku V2 node REST API
+  version: 1.0.0
+  contact:
+    name: VAC Team
+    url: https://forum.vac.dev/
+
+tags:
+  - name: lightpush
+    description: Lightpush REST API for WakuV2 node
+
+paths:
+  /lightpush/v1/message:
+    post:
+      summary: Request a message relay from a LightPush service provider
+      description: Push a message to be relayed on a PubSub topic.
+      operationId: postMessagesToPubsubTopic
+      tags:
+        - lightpush
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PushRequest'
```

@NagyZoltanPeter shouldnt the `requestyBody` be required?
```suggestion
      requestBody:
        required: true
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/PushRequest'
```

Also, in the schema, what do you think of adding an optional `peerID` so the user can specify which peerID to use to publish the messages, instead of making it random? and if the user does not specify this field, then just use the current behavior.

---

I think so, request body is mandatory.

About peerID, currently it is not defined for the interface.
At least in wakunode, when enabling lightpush client interface the operator must define the target peer.

Please check out this: https://github.com/waku-org/nwaku/issues/2136
This is actually about this to improve.